### PR TITLE
Fix: Tests and Fix for user enumeration bug

### PR DESCRIPTION
**Pull request by PvtSec**
## Changes made:

* Add test coverage for invalid logins
* Add fix for the issue: [user enumeration bug via login](https://github.com/hcengineering/platform/issues/5202)

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjEyMjgwOTY4ZWY5NjM0ZDNmMDQyNGEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.akbx1URMWw_g1h_pPK7exKVHdcYmdahs65zlbUwj6QU">Huly&reg;: <b>UBERF-6397</b></a></sub>

### CODE REVIEW

**Review by haiodo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,18 +24,24 @@ This project offers a convenient method to host Huly using `docker`, designed fo
 
 ## Table of Content
 
-- [Pre-requisites](#pre-requisites)
-- [Fast start](#fast-start)
-- [Installation](#installation)
-- [Build and run](#build-and-run)
-- Development mode
+- [Huly Platform](#huly-platform)
```
Comment (haiodo) [inline README.md]: Let's remove this changes from this PR. Thanks.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -265,7 +265,7 @@ jobs:
           path: ./allure-report/
       - name: Deploy report to Github Pages
         if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v') }}
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@v3.9.3
```
Comment (haiodo) [inline .github/workflows/main.yml]: Let's remove this file changes in this PR.

File: server/account/src/index.ts [hunk 3]
```diff
@@ -234,13 +234,13 @@ function toAccountInfo (account: Account): AccountInfo {
 async function getAccountInfo (ctx: MeasureContext, db: Db, email: string, password: string): Promise<AccountInfo> {
   const account = await getAccount(db, email)
   if (account === null) {
-    throw new PlatformError(new Status(Severity.ERROR, platform.status.AccountNotFound, { account: email }))
```
Comment (haiodo) [inline server/account/src/index.ts:245]: I think we still need log states, InvalidPassword into context logger and pass them into analytics, so we could understand and track a problems.

**Review by haiodo — CHANGES_REQUESTED** [review]
Hi @PvtSec 
Thanks for a contribution, but some work is still to be done accept changes. Thanks.

**Comment by PvtSec** [comment]
Hi @haiodo ,

- Reverted changes
- Keeps logging for older states
- Resolved all comments.

Ready for a new review

**Pull request closed without merging**